/**:
  ros__parameters:
    max_x: 10.0
    min_x: -5.0
    max_y: 5.0
    min_y: -5.0
    filter_target_label:
      UNKNOWN: true
      CAR: false
      TRUCK: false
      BUS: false
      TRAILER: false
      MOTORCYCLE: false
      BICYCLE: false
      PEDESTRIAN: false
    nearest_points_num: 5
    normal_threshold: 0.5